ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-rollback-nightly.yaml

```diff
@@ -41,7 +41,7 @@ releases:
     release:
       architecture: amd64
       channel: candidate
-      version: 4.17.0-rc.0
+      version: "4.17"
```

Comment:
I don't know the original purpose to use 4.17.0-rc.0. Need other reviewer's confirmation

Reply:
I think it's better to avoid hard-coding to a specific image, but yeah I'll make sure to tag liangxia

Reply:
The specific version requirement comes from OTA team.  Could you help confirm if the change is expected, CC @jiajliu @shellyyang1989

Reply:
AFAIK, hard-coding should be by purpose for nightly rollback test. It will be updated to 4.17.0 once 4.17 release.

Reply:
@liangxia @jiajliu 

I've reverted the change for now and will confirm with SD later. If necessary, I'll raise another PR.

Reply:
Would you mind to update the PR description to make it consistent ?

Reply:
@liangxia Updated